news/2019-02-22-acknowledging-community-contributors.md

```diff
@@ -0,0 +1,211 @@
+
+We could say many things about HappyStick, but foremost among them is how thankful we are for his continued contribution.
+
+### juankristal
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/juankristal.jpg)](https://osu.ppy.sh/users/443656)
+
+Something of a living meme among the osu! tournament scene, juankristal has been perpetually blamed for everything and anything that has gone both wrong and right for countless years.
+
+For once, we are delighted to firmly pin the blame on him for the continuation of the osu! World Cup series as a whole, stepping up to lead the INTENSE organization required to schedule, oversee and generally maintain multiple multinational tournaments.
+
+The workload involved in running the osu! World Cups is *immense*, perhaps far greater than most people realize, and juankristal has not only stepped up himself to do it out of a sheer desire to see it done, but has *excelled* at it, allowing the series to continue without so much as a blip.
+
+### WalterToro
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/WalterToro.jpg)](https://osu.ppy.sh/users/5281416)
+
+A mainstay force within the Global Moderation Team, WalterToro is not your ordinary chat moderator. He is *everywhere*, in the osu!wiki scene, in the tournaments scene, and wildly active in the Spanish community to say the least.
+
+From refereeing matches to refereeing people, we are intensely thankful for WalterToro's incredible work ethic and tireless drive to do what needs to be done.
+
+### clayton
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/clayton.jpg)](https://osu.ppy.sh/users/3666350)
+
+From his involvement in the development of the new website, to reviewing countless improvements and additions to the osu!wiki and various newsposts, to assisting with the maintenance and general day-to-day tasks with Project Loved and much more, clayton has done almost everything there is to do, and continues to keep doing it.
+
+Getting things done is what keeps osu! alive and moving, and there is perhaps no better example of this than clayton in recent times. We are supremely thankful for his efforts over the past few years.
+
+### VeilStar
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/VeilStar.jpg)](https://osu.ppy.sh/users/4255720)
+
+A formerly gigantic presence in the forum technical support team, VeilStar has been responsible for the resolution of literally *thousands* of technical issues over the years and has helped similar amounts of people enjoy osu! to its fullest.
+
+Having since departed the team and osu! in general to pursue things elsewhere, we'd like to commit just how thankful we are for his assistance over the years.
+
+### Stefan
+
+[![](/wiki/shared/news/2019-02-22-acknowledging-community-contributors/people/Stefan.jpg)](https://osu.ppy.sh/users/626907)
```

Comment:
Current avatar is different than one pictured in banner. Not sure if it matters at all but pointing it out anyways.

Reply:
oh my god he's changed it since yesterday AHHHHHH